### ISSUE TITLE
Width is scaled by the total sum of values?

### ISSUE THREAD

**Issue by stnever** [tool3/chartscii#72]
When calculating the bar length, we are always dividing by `this.maxCount` (even on non-percentage charts). The result is that the chart width gets smaller the more rows you have (even if they have about the same value).

https://github.com/tool3/chartscii/blob/d1adae5b3c10e42197647c4f53acf8ea6d80afc0/index.js#L178

To reproduce:
```javascript
const Chartscii = require('chartscii');

const data1 = [5,10,15,5,10,15];
const chart1 = new Chartscii(data1, {width: 100});
console.log(chart1.create());

const data2 = [5,10,15];
const chart2 = new Chartscii(data2, {width: 100});
console.log(chart2.create());
```
This outputs:

![image](https://user-images.githubusercontent.com/342285/113785432-8106af00-970d-11eb-93ac-a3ac01e490e1.png)

Note that the second chart is wider than the first, even though `{width:100}` is used on both, and they both contain the same min/max values. This is because `this.maxCount` is 60 in the first chart and 30 in the second, so the division in the source line above yields a wider bar.

I expected both charts to have the same width.

**Comment by tool3** [comment]
@stnever thank you so much for opening this issue. this thing has been in the back of my mind, for a very long time.

the repro now produces the expected result.

```javascript
const Chartscii = require('chartscii');

const data1 = [5,10,15,5,10,15];
const chart1 = new Chartscii(data1, {width: 100});
console.log(chart1.create());

const data2 = [5,10,15];
const chart2 = new Chartscii(data2, {width: 100});
console.log(chart2.create());
```

```text
15 ╢████████████████████████████████████████████████████████████████████████████████████████████████████
10 ╢███████████████████████████████████████████████████████████████████
 5 ╢█████████████████████████████████
15 ╢████████████████████████████████████████████████████████████████████████████████████████████████████
10 ╢███████████████████████████████████████████████████████████████████
 5 ╢█████████████████████████████████
   ╚════════════════════════════════════════════════════════════════════════════════════════════════════
15 ╢████████████████████████████████████████████████████████████████████████████████████████████████████
10 ╢███████████████████████████████████████████████████████████████████
 5 ╢█████████████████████████████████
   ╚════════════════════════════════════════════════════════════════════════════════════════════════════
```

scaling is still being done through `width`, so in your repro example, if I set `width: 50` on the second chart, the output will look like this:

```text
15 ╢████████████████████████████████████████████████████████████████████████████████████████████████████
10 ╢███████████████████████████████████████████████████████████████████
 5 ╢█████████████████████████████████
15 ╢████████████████████████████████████████████████████████████████████████████████████████████████████
10 ╢███████████████████████████████████████████████████████████████████
 5 ╢█████████████████████████████████
   ╚════════════════════════════════════════════════════════════════════════════════════════════════════
15 ╢██████████████████████████████████████████████████
10 ╢█████████████████████████████████
 5 ╢█████████████████
   ╚══════════════════════════════════════════════════
```

dividing by `maxCount` was wrong, so I'm glad you opened this issue - please let me know if this resolves it.

**Comment by tool3** [comment]
fixed in `chartscii@1.2.5`
